native/src/App.tsx

```diff
@@ -51,6 +52,9 @@ const linking: LinkingOptions = {
 }
 const dataContainer: DataContainer = new DefaultDataContainer()
 const store: Store<StateType, StoreActionType> = createReduxStore(dataContainer)
+defaultRequestOptions.headers = {
+  'User-Agent': userAgent
+}
```

Comment:
I think a `Request.setUserAgent()` function would be cleaner and less error prone.